doc/releases/release-notes-2.1.rst

```diff
@@ -335,7 +364,34 @@ Networking
 Bluetooth
 *********
 
-* TBD
+* Host:
+
+  * Reworked the Host transmission path to improve memory footprint and remove potential deadlocks
+  * Document HCI errors for connected callback
+  * GATT: Added a ``bt_gatt_is_subscribed()`` function to check if attribute has been subscribed
+  * GATT: Added an initializer for GATT CCC structures
+  * HCI: Added a function to get the connection handle of a connection
+  * Added ability to load CCC settings on demand to reduce memory usage
+  * Made the time to run slave connection parameters update procedure configurable
+  * Folded consecutive calls to bt_rand into one to reduce overhead
+  * Added key displacement feature for key storage
+  * Reduced severity of unavoidable warnings
+  * Added support C++20 designated initializers
+  * Mesh: Add the model extension concept as described in the Mesh Profile Specification
```

Comment:
I'd add here "Mesh: Added support for acting as a Provisioner"

Reply:
ok